### ISSUE TITLE
Lots of UI components are unresponsive when using Sprites

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/gas-properties#190]
In #157, I converted from CanvasNode to Sprites, as @jonathanolson felt this was an important improvement.   The main commit was https://github.com/phetsims/gas-properties/commit/f32a2cfdc96a9d974032be391c717ec8c05c3fc7.

That change, or something related to Sprites that has broken since the change was made, has broken much of the Gas Properties UI.  From https://github.com/phetsims/sun/issues/676#issuecomment-765160770, the following UI components are now unresponsive:

> Ideal screen:
> * all of the "Hold Constant" radio buttons
>
> Explore screen
> * thermometer combo box
> * all of the checkboxes
> 
> Energy Screen
> * Average Speed accordion box expand/collapse button
> * Particles accordion box expand/collapse button
> * Checkboxes in the Speed accordion box
> * handle on the top of the box
> * checkboxes in the top-right panel
> * thermometer combo box
> 
> ...
> All of the things that are unresponsive seem to be in the top part of the screen.

I tried reverting to CanvasNode by checking out a876535219709444eef057857fc895afaba29ae3, the commit before I changed to Sprites. Then I replaced Utils.toFixed with Number.toFixed so the sim would run. All of the problems with UI components go away.

@jonathanolson can you please look at this with high priority?

**Comment by pixelzoom** [comment]
I also tried setting `webgl: false` in gas-properties-main.js, so that things would fall back to Canvas. That didn't resolve the problem.  So it may be a problem that's independent of renderer?

**Comment by pixelzoom** [comment]
ParticlesNode extends Sprites, and in the above commit I've set ParticlesNode to `pickable: false`.  This fixes the problems. 

I guess it's possible that I didn't notice this problem when I converted to Sprites.  @jonathanolson do you think this is what happened? Would you expect `pickable:false` to be necessary in this case?  Or might something have changed that caused this to break?

**Comment by pixelzoom** [comment]
Discussed with @jonathanolson on Slack, see thread below.

This was caused by a new problem that was introduced, see https://github.com/phetsims/sun/issues/676#issuecomment-767296291. Basically, anything that doesn't explicitly set `pickable: false` is behaving like `pickable: true` and swallowing events.  So in this sim,  Sprites was blocking events from reaching UI components. 

The `pickable: false` workaround is OK to use longterm, so closing this issue.

<details>
<summary>Slack thread</summary>

Jonathan Olson  11:24 AM
Looks like it's mainly the "listener on every ScreenView is making way more stuff intercept events" thing, but making it pickable helps?

Chris Malley  11:26 AM
I'm not familiar with the "listener on every ScreenView is making way more stuff intercept events" thing.  Is there a GitHub issue for that?

Jonathan Olson  11:26 AM
Referring to https://github.com/phetsims/sun/issues/676#issuecomment-767296291

Chris Malley  11:27 AM
And is that a new problem that suddenly caused this problem for Gas Properties + Sprites?
... or did I blow it when I converted Gas Properties from CanvasNode to Sprites?

Jonathan Olson  11:27 AM
It seems to be causing issues in tons of sims. So anything that wasn't specifically "pickable:false" but didn't have input listeners before is NOW acting like it's "pickable:true".  Because now basically everything "has an input listener on it"

Chris Malley  11:27 AM
So yes, it's a new problem?

Jonathan Olson  11:27 AM
yes

Chris Malley  11:28 AM
OK. Good to know I'm not losing my mind :). .Because I couldn't figure out how I could have missed the fact that Gas Properties was so thoroughly broken.  So I'll note this discussion in https://github.com/phetsims/gas-properties/issues/190, and I'm OK closing with the pickable: false change for my Sprites subclass. Cool?

Jonathan Olson  11:29 AM
Sounds great!

</details>
